Assets/!/Scripts/PlayerAudio.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.IO;
+using utils;
+
+public class PlayerAudio : MonoBehaviour
+{
+    public AudioSource audioSource;
+    public List<Audio> audioPlayer = new List<Audio>();
```

Comment:
Use arrays since we wont update its sice on runtime

Reply:
this change always leads problems of null check, is it possible we keep it as List